### PR TITLE
Flyer test

### PR DESCRIPTION
**Pull request by JunAishima**
fixes towards getting LSDC running with the NYX flyer object

### CODE REVIEW

File: daq_lib.py [hunk 1]
```diff
@@ -626,39 +626,40 @@ def collectData(currentRequest):
...
-        if (daq_utils.beamline == "amx"):                                            
-          visitName = daq_utils.getVisitName()
-          if (not os.path.exists(visitName + "/fast_dp_dir")):
-            os.system("killall -KILL loop-fdp-dple-populate")
-            os.system("cd " + visitName + ";${LSDCHOME}/bin/loop-fdp-dple-populate.sh&")
-        os.system(comm_s)
-      if (reqObj["xia2"]):
-        comm_s = f"ssh -q xf17id2-srv1 \"{os.environ['MXPROCESSINGSCRIPTSDIR']}xia2.sh {currentRequest['uid']} \"&"
-        os.system(comm_s)
+    if daq_utils.beamline != "nyx":
```
Comment (RobertSchaffer1) [inline daq_lib.py:629]: Do we have to put in something equivalent for nyx?

File: gov_lib.py [hunk 2]
```diff
@@ -32,3 +32,6 @@ def waitGov(status, timeout=GOVERNOR_TIMEOUT):
     message = 'Governor Timeout!'
     logger.error(message)
     #gui_message(message)
+
```
[the remark this answers is not in the document]
Comment (JunAishima) [reply gov_lib.py:35]: `gui_message` is a mechanism to send strings to the GUI in a box that the user needs to acknowledge. I think I commented out imports from `daq_lib` when there were some circular imports

File: daq_lib.py (hunk 1 above)
Comment (JunAishima) [reply daq_lib.py:629]: not for now - while I don't encourage it, Kevin insists on doing the processing his own way.

we should bring it up with him and see whether he wants to piggyback on the current AMX/FMX code. he has not wanted to in the past

File: gov_lib.py (hunk 2 above)
[the remark this answers is not in the document]
Comment (RobertSchaffer1) [reply gov_lib.py:35]: Sorry, I meant the code below that was added, the do-nothing function. Just to satisfy an interface, or are we planning to expand on that later?

**Review by RobertSchaffer1 — APPROVED** [review]
Looks good, I would love to know what that function is doing though - maybe we can figure it out tomorrow.

**Pull request merged**